VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -93,6 +96,7 @@ qMRMLVirtualRealityViewPrivate::qMRMLVirtualRealityViewPrivate(qMRMLVirtualReali
   , CamerasLogic(NULL)
 {
   this->MRMLVirtualRealityViewNode = 0;
+  this->HomeWidget = new qMRMLVirtualRealityHomeWidget(q_ptr);
```

Comment:
How do you access this one? I cannot see a getter function in the view.

Reply:
That line just initializes the member variable HomeWidget, then I access the member directly through a D-pointer in the registerModule function. Should I make a getter function and access that instead?